skimage/measure/simple_metrics.py

```diff
@@ -142,3 +146,47 @@ def compare_psnr(im_true, im_test, data_range=None):
 
     err = compare_mse(im_true, im_test)
     return 10 * np.log10((data_range ** 2) / err)
+
+
+def enhancement_measure(image: np.ndarray,
+                        size: int = 3) -> float:
+    """
+        The image enhancement measure called EME based on [1]_.
+
+        Parameters
+        ----------
+        image : array
+            Input image of which the quality should be assessed.
+            Can be either 3-channel RGB or 1-channel grayscale.
+            The function converts pixel intensities into floats by default.
+        size : int
+            Size of the window. Default value is 3.
+
+        Returns
+        -------
+        eme : float
+            The number describing image quality.
+
+        References
+        ----------
+        .. [1] Agaian, Sos S., Karen Panetta, and Artyom M. Grigoryan.
+               "A new measure of image enhancement."
+               IASTED International Conference on Signal Processing
+               & Communication. Citeseer, 2000.
+        Examples
+        --------
+        >>> from skimage.data import camera
+        >>> from skimage.exposure import equalize_hist
+        >>> img = camera()
+        >>> quality(img) # before
+            0.9096745071475523
+        >>> quality(equalize_hist(img))
+            1.299327371881219
+
+    """
+    image = img_as_float(image)
+    eme = np.zeros_like(image)
+    eme = np.divide(maximum_filter(image, tuple([size]*len(image.shape))) + 1,
+                    minimum_filter(image, tuple([size]*len(image.shape))) + 1)
```

Comment:
Sorry, what I meant earlier is that [`maximum_filter`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.ndimage.maximum_filter.html) already supports the parameter `size` directly. E.g.

```python
eme = np.divide(maximum_filter(image, size=size)) + 1,
                minimum_filter(image, size=size)) + 1)
```
Or am I reading that wrong?

Reply:
I think you're right. I was using the `footprint` parameter, because I got the impression from the docs that it is necessary for proper shape of the filter. I tested with just using `size' and it seems to work as well.